addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -17,7 +17,8 @@ def setUp(self):
         self.env.company.street = "test"
         self.env.company.phone = "12345"
 
-        self.partner_id = self.env['res.partner'].create({"name": "l10ntest", "l10n_id_pkp": True, "l10n_id_kode_transaksi": "01", "l10n_id_nik": "12345"})
+        self.partner_id = self.env['res.partner'].create({"name": "l10ntest", "l10n_id_pkp": True, "l10n_id_kode_transaksi": "01", "l10n_id_nik": "12345", "vat": "000000000000000"})
+        self.partner_id_vat = self.env['res.partner'].create({"name": "l10ntest3", "l10n_id_pkp": True, "l10n_id_kode_transaksi": "01", "l10n_id_nik": "67890", "vat": "010000000000000"})
```

Comment:
You should use partner_id_vat in one of the two invoices (or add a new test for it)